### DISCUSSION TITLE
R 5.0.1 Sunbird ED - Mobile - Circle QA Sign Off

### DISCUSSION THREAD

**Discussion by PoornimaRajesh** [Sunbird-Ed/Community#194] [General Discussion]
Hello All,

This is Consolidated circle QA Sign off for the release 5.0.1.0 for Sunbird ED- Mobile Building block.

Environment: Staging
Testing Status: Completed
Defects Dashboard: NA

Execution Sheet: [Link](https://docs.google.com/spreadsheets/d/1pP1eRNXYRNTiJe0BYZ2BowHRJ9ygCi6VKIRSJ2cgFMM/edit#gid=1581973510)

Scope : [Link](https://project-sunbird.atlassian.net/jira/software/c/projects/ED/boards/103/backlog?selectedIssue=ED-76&quickFilter=351&issueLimit=100)

Circle Sign Off Template: [Link](https://docs.google.com/spreadsheets/d/1jail-mLcIooaYbRIIRWVeFmXfw90oNoY_g_SYayp2Wg/edit#gid=1621611375)

cc: @luckynvdu @sharathkashyap @swayangjit @chitranshu-keshav
@anunagaraj @ShailandraOjha

**Comment by anunagaraj** [comment]
@anisha147 Microsite is not updated with the R5.0.1 Tickets which have been signed off for Sunbird Ed. Kindly update the Microsite.

CC @luckynvdu @sharathkashyap @ShailandraOjha @chitranshu-keshav
